reviews/2021-k3s-annual.md

```diff
@@ -127,35 +101,104 @@ Currently, the project has 11 maintainers, all with SUSE:
 
 ### What do you know about adoption, and how has this changed since your last review / since you joined Sandbox? If you can list companies that are end users of your project, please do so. (Feel free to link to an existing ADOPTERS file if appropriate.)
 
-Some adoption highlights from the last year are:
+Based on issues filed on GitHub and interactions on Slack, K3s continues to see a healthy adoption from many different types of users from hobbyists to large organizations. However, we haven't begun to formally track adopters or ask their permission to list them as such. We will start to do so now.
+
+One adoption highlight is Civo's use of K3s to power its Kubernetes cloud offering, see: https://www.civo.com/kubernetes.
+
+K3s is also used as a building block for other projects and products. Here are a few that are independent of K3s, but leverage or build on top of it:
+* [k3d](https://github.com/rancher/k3d)
+* [k3sup](https://github.com/alexellis/k3sup)
+* [Rancher Desktop](https://rancherdesktop.io/)
+
+We also track downloads of our binaries from GitHub (the primary way to install K3s). Over the past year, our cumulative, all-time number of downloads grew from right around 1.2 million to 2.7 million. Note that this number does not represent unique users. It is simply the number of times the k3s binaries have been downloaded.
 
-1. [Civo Kubernetes](https://www.civo.com/kubernetes)
-2. [Northflank](https://northflank.com/)
 
 ### How has the project performed against its goals since the last review? (We won't penalize you if your goals changed for good reasons.)
 
-- [x] SELinux Policy
-- [ ] Dual Stack Networking
+This is our first annual review, so here is a list of our accomplishments since joining the CNCF. When we submitted for inclusion in the CNCF, we outlined the following roadmap items:
+
+- Improving stability and support across operating systems and architectures
+  - _We did achieve this by improving our ARM builds and supporting more operating systems_
+- Enhancing security through SELinux support and CIS compliance
+  - _We did introduce SELinux support and published a guide evaluating K3s against the CIS Kubernetes benchmark_
+- Developing HA solutions appropriate for small edge clusters where even three nodes are cost prohibitive
+  - _We did not complete this. We chose instead to focus on embedded etcd support, which meets the use cases we were concerned about._
+- Improving support for Kubernetes cloud providers
+  - _We chose not to do this. Cloud providers are moving out of tree and K3s supports that mode properly. We decided that was enough._
+- Improved support for database backends, including embedded etcd
+  - _We achieved this. K3s now supports an "embedded etcd" mode where it will launch and manage etcd, including snapshot and restore functionality._
```

Comment:
Maybe mention the improvements we've made to kine? Transaction-locked pure-sql compaction, metrics support, stubbed out GRPC services so that it works out of the box with kubeadm, etc.